pkg/apply/processor/scale.go

```diff
@@ -150,8 +150,8 @@ func (c *ScaleProcessor) DeleteCheck(cluster *v2.Cluster) error {
 	logger.Info("Executing pipeline DeleteCheck in ScaleProcessor.")
 	var ips []string
 	ips = append(ips, cluster.GetMaster0IPAndPort())
-	ips = append(ips, c.MastersToDelete...)
-	ips = append(ips, c.NodesToDelete...)
+	//ips = append(ips, c.MastersToDelete...)
```

Comment:
Will deleting this cause some problems？ such as on bare metal

Reply:
Actually i don't know the meaning of this checking exactly,  in my view, ensuring all these ssh-connect is simply used for cleaning the nodes to be deleted, which is optional considering cloud provider, but i'm not sure..